frontend/src/app/modules/global_search/global-search-input.component.ts

```diff
@@ -289,6 +311,10 @@ export class GlobalSearchInputComponent implements OnInit, OnDestroy {
     this.$element.find('.ui-autocomplete--loading').hide();
   }
 
+  private get isMobile():boolean {
```

Comment:
We probably will need this method in more places in future. What about extracting that into a separate service? So it could look like `Device.isMobile()` for example.